lib/dynamic_widget.dart

```diff
@@ -145,8 +169,12 @@ class DynamicWidgetBuilder {
   }
 
   static Widget? buildFromMap(Map<String, dynamic>? map,
-      BuildContext buildContext, EventListener? listener) {
+      BuildContext buildContext, EventListener? listener,
+      {Widget? child}) {
     initDefaultParsersIfNess();
+    if (child != null) {
+      return child;
```

Comment:
who is going to pass child here?

Reply:
I'll remove it, was adding it to add a child valueListenableBuilder